purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -245,14 +245,28 @@ fun Purchases.identifyWith(
  * @param [onError] Will be called after the call has completed with an error.
  */
 @Suppress("unused")
-fun Purchases.logInWith(
+internal fun Purchases.logInWith(
```

Comment:
although this is in `develop`, it has luckily not made it into a release yet, so we should be good.